README.md

```
@@ -311,6 +311,274 @@ print(json_str)
+- 🎯 **Default values** → pre-set starting data  
+- ✅ **Constraints** → like `notnull` or `notblank`  
+
+💡 **In short:** Everything needed to recreate your class exactly as you defined it — without touching a single line of code again.  
+
+---
+
+### 📤 Exporting Configurations  
+
+```python
+Config.export_config()
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)